source/rust_verify/src/fn_call_to_vir.rs

```diff
@@ -143,8 +143,10 @@ pub(crate) fn fn_call_to_vir<'tcx>(
     }
 
     let f_attrs = bctx.ctxt.tcx.get_attrs_unchecked(f);
-    let f_vattrs = get_verifier_attrs(f_attrs, Some(&mut *bctx.ctxt.diagnostics.borrow_mut()))?;
-    if f_vattrs.internal_get_field_many_variants {
+    if crate::attributes::is_get_field_many_variants(
+        f_attrs,
+        Some(&mut *bctx.ctxt.diagnostics.borrow_mut()),
+    )? {
         return Err(vir::messages::error(
```

Comment:
Can you explain why this change is necessary: i.e. why you cannot `get_verifier_attrs`  via `bctx.ctxt` here?

Reply:
Sometimes it would examine a function in the rust standard lib that we support via external_fn_specification.

Reply:
Can you please add a comment to explain that?